### DISCUSSION TITLE
Startup slow when using different Internet Connections for IN and OUT

### DISCUSSION THREAD

**Discussion by DrakeHamString** [nextcloud/all-in-one#4529] [Questions] [answered]
### Steps to reproduce
1. Having two Internet Connections (ISPs) for failsave operation, managed by a router (who does NATing)
2. DNS record points to IPS 1 for incoming access; default internet / outgoing traffic goes trough IPS 2
3. Setting up AIO

### Expected behavior 

Normal AIO Operation, fast start of containers.
Snappy operation.

### Actual behavior 

Starting of ''nextcloud-aio-nextcloud'' takes about 30 minutes.
The normal setup works fine, the slow start appears when doing the "normal" start of the container after setup is done.
When the container is started, the web interface is very slow and laggy. There are minutes of wait time until the website loads.
But its reachable.

### Host OS 

Linux

#### Nextcloud AIO version 

v7.13.1 (current behaviour expected in latest vesion too)

#### Current channel 

latest

#### Other valuable info 

It seems that there are internal checks or timeouts which are causing the slow start. Where it checks if ingoing and outgoing traffic ways are matching.
It would be nice to be able to disable those checks or at least let the logs tell you whats wrong. Here was nothing telling me about any issue. It was just slow, there were no error or info messages.

I already posted this in the forum before opening the issue on GitHub:
https://help.nextcloud.com/t/nextcloud-aio-taking-extemely-long-to-start-debugging-options/185956

After setting all traffic through one ISP did solve the issue for me. But the problem still remains when there is an actual need to run it though two ISPs.

I also switched the IPS's roles to reproduce the issue the other way arround - yes, the issue was there again.

**Comment by DrakeHamString** [comment]
Funny thing I also noticed:
At the moments when the DNS record was not propagated yet and the NC AIO didnt know the IP address behind the DNS name, the containers did start fast. But right at this moment the DNS record was present, the containers became slow again.

**Comment by szaimen** [comment, marked as answer]
Hi, I fear this is nothing the AIO project controls. You need to fix your networking to use the correct dns

Maybe actually this helps? https://github.com/nextcloud/all-in-one/issues/4519

**Comment by DrakeHamString** [comment]
Actually that described network config is intended and I prefer it to be that way. 

The issue here is, that the container didnt tell me that it has a problem with it. Or that there is a problem in general.

tl;dr:
Would be nice to make AIO to work with two internet connections for IN and OUT respectively
